plugins/package-managers/bundler/src/main/kotlin/Bundler.kt

```diff
@@ -453,13 +453,17 @@ internal data class GemSpec(
                 RemoteArtifact.EMPTY
             }
 
+            val description = sequenceOf(node["description"], node["info"]).mapNotNull { text ->
+                text?.textValue()?.takeIf { it.isNotEmpty() }
+            }.firstOrNull()
```

Comment:
Would it make sense to have a code comment explaining this fallback?

Furthermore, would it make sense to adjust the fallback to: description -> summary -> info ?